modules/ui/src/app/pages/risk-assessment/risk-assessment.store.ts

```diff
@@ -48,7 +49,48 @@ export class RiskAssessmentStore extends ComponentStore<AppComponentState> {
       })
     );
   });
-  constructor(private testRunService: TestRunService) {
+
+  deleteProfile = this.effect<string>(trigger$ => {
+    return trigger$.pipe(
+      exhaustMap((name: string) => {
+        return this.testRunService.deleteProfile(name).pipe(
+          withLatestFrom(this.profiles$),
+          tap(([remove, current]) => {
+            if (remove) {
+              this.removeProfile(name, current);
+            }
+          })
+        );
+      })
+    );
+  });
+
+  setFocus = this.effect<{ nextItem: HTMLElement; firstItem: HTMLElement }>(
+    trigger$ => {
+      return trigger$.pipe(
+        withLatestFrom(this.profiles$),
+        tap(([{ nextItem, firstItem }, profiles]) => {
+          if (nextItem) {
+            this.focusManagerService.focusFirstElementInContainer(nextItem);
+          } else if (profiles.length > 1) {
```

Comment:
I think check first item (as it's done with next item on like 73), should work as well and there is no need to check the length